flask/cli.py

```diff
@@ -398,6 +398,8 @@ def run_command(info, host, port, reload, debugger, eager_loading,
     The reloader and debugger are by default enabled if the debug flag of
     Flask is enabled and disabled otherwise.
     """
+    import sys
```

Comment:
Sure, but the fact is that the behaviour between python -m flask run and flask run is inconsistent. Not an ideal fix, but then again not every flask app should need a setup.py, if it not being distributed via PyPI.